### PR TITLE
Feat/update

### PR DESCRIPTION
**Pull request by tvmaasjazz**
Not sure if you want there to be an example in here of some functionality but this serves as the skeleton for how I've been structuring these typedi projects

### CODE REVIEW

File: config/default.yml [hunk 1]
```diff
@@ -1,8 +1,18 @@
-logger:
-    level: debug
+# logger:
```
Comment (jpinkster) [inline config/default.yml]: commented?

File: config/default.yml [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
-    fileOutput: 'users.csv'
+email:
+    noReplyEmailAddress: 'dev@meetmaestro.com'
+    sendTo: ['placeholder@placeholder.com']
+    send: true
+    sendgrid:
+        apiKey: 'PLACEHOLDER'
+
+raven:
+    dsn: ''
```
Comment (jpinkster) [inline config/default.yml:5]: you should put in PLACEHOLDER for placeholder keys .. intentionally "break" it so the devs use it

File: package.json [hunk 3]
```diff
@@ -1,52 +1,59 @@
...
+        "John Pinkster <jpinkster>",
+        "Tyler Vander Maas <tvmaasjazz>"
+    ],
+    "license": "ISC",
+    "bugs": {
+        "url": "https://github.com/TeamHive/typedi-seed/issues"
+    },
+    "homepage": "https://github.com/TeamHive/typedi-seed#readme",
+    "dependencies": {
+        "@teamhive/typedi-common": "file:../typedi-common/teamhive-typedi-common-1.0.0.tgz",
```
Comment (jpinkster) [inline package.json]: say what?

File: pull_rerquest_template.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (jpinkster) [inline pull_rerquest_template.md]: lol i wondered about this .. how will we maintain a master version of this since this is outdated to the others

**Comment by tvmaasjazz** [comment]
Updated to remove emails. This will work with the updated typedi-common in the open PR Feat/typedi common

**Pull request merged**